### PR TITLE
Fix drive code examples

### PR DESCRIPTION
**Pull request by RishabRao**
Java version fixed. Cpp still needs fixing 

https://docs.wpilib.org/en/latest/docs/software/actuators/wpi-drive-classes.html?highlight=spark

#214

### CODE REVIEW

File: source/docs/software/actuators/wpi-drive-classes.rst [hunk 1]
```diff
@@ -180,10 +180,16 @@ as so:
 
     .. code-tab:: java
 
-        Spark m_left = new Spark(1);
-        m_left.setInverted(true); // if you want to invert motor outputs, you must do so here
-        Spark m_right = new Spark(2);
-        DifferentialDrive m_drive = new DifferentialDrive(m_left, m_right);
+        public class Robot
+        {
```
Comment (calcmogul) [inline source/docs/software/actuators/wpi-drive-classes.rst]: Please put braces on the same line as the class/function declaration.

File: source/docs/software/actuators/wpi-drive-classes.rst [hunk 2]
```diff
@@ -211,23 +216,24 @@ constructor (it takes an arbitrary number of inputs).
...
             SpeedControllerGroup m_left = new SpeedControllerGroup(m_frontLeft, m_rearLeft);
 
             Spark m_frontRight = new Spark(3);
             Spark m_rearRight = new Spark(4);
             SpeedControllerGroup m_right = new SpeedControllerGroup(m_frontRight, m_rearRight);
             DifferentialDrive m_drive = new DifferentialDrive(m_left, m_right);
 
-    .. code-tab:: c++
+            public void robotInit()
+            {
```
Comment (AustinShalit) [inline source/docs/software/actuators/wpi-drive-classes.rst]: Fix brackets to follow style guide

**Comment by RishabRao** [comment]
> Do you need support for the C++ version?

yes please

**Comment by sciencewhiz** [comment]
@RishabRao I updated the C++ examples

**Pull request merged**